### ISSUE TITLE
add shortcut to create new Aurelia gist

### ISSUE THREAD

**Issue by jdanyow** [gist-run/gist-run#14]
cc @egeland

**Comment by egeland** [comment]
Yep, I just thought it'd be cool to have a "set up Aurelia" button, so gist-run will become the go-to place to demonstrate Aurelia.. :smile:

**Comment by opcodewriter** [comment]
i was thinking about having "New" button similar to a dropdown button. the difference is it would display a menu on hover. the menu can show for now only "new Aurelia gist". the button will still create same gist as today on click. too complicated maybe? thoughts?

**Comment by PWKad** [comment]
For the next time I come here looking for a link - https://gist.run/?id=883568d7c8e2add9ed393150100f4744

**Comment by ZoolWay** [comment]
Would that also be possible with TypeScript?

**Comment by Reverbe** [comment]
Also wondering if TypeScript can be supported

**Comment by plwalters** [comment]
TypeScript could possibly be supported but I think it would be a fundamental change to gist.run so I'm not sure it will be supported any time soon.  I think a pull request would be accepted though if you wanted to add TypeScript support.

**Comment by jdanyow** [comment]
I'm working on a switch from the ace editor to the monaco editor. Part of that work is built-in babel and typescript support.

**Comment by vegarringdal** [comment]
> I'm working on a switch from the ace editor to the monaco editor. Part of that work is built-in babel and typescript support.

@jdanyow Any news on updates ?
